src/state/farms/fetchPublicFarmData.ts

```diff
@@ -15,6 +15,7 @@ type PublicFarmData = {
   lpTotalSupply?: SerializedBigNumber
   tokenPriceVsQuote?: SerializedBigNumber
   poolWeight?: SerializedBigNumber
+  multiplier?: string
```

Comment:
what is this :D

Reply:
it is returned here but was missing in type, I guess @ChefHutch just fixed it while he's changing this file 🙂

Reply:
Ah yes true, I thought it is related to timeout thing